grommet/grommet
# Adjust onReference TS definition

#### What does this PR do?
This PR adjusts the type definition for onReference to be `(reference: string) => void`.

#### Where should the reviewer start?
src/js/components/Calendar/index.d.ts

#### What testing has been done on this PR?
This has been tested in a local TS project where a Calendar storybook example (Dual) was copied into my local project to check that the onReference handler had no flags. It did not, and that app compiled successfully.

#### How should this be manually tested?
Copy a storybook example containing onReference into a local TS project and make sure there are no errors in your editor.

#### Any background context you want to provide?
This line in Calendar.js shows how onReference is used. It accepts a reference which is a string, and converts it to an ISO string.
```
if (onReference) {
          onReference(reference.toISOString());
        }
```
 I used this to know how to create the type definition.

#### What are the relevant issues?
#3165 

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
No.

#### Should this PR be mentioned in the release notes?
No.

#### Is this change backwards compatible or is it a breaking change?
Yes, this is backwards compatible.
